### PR TITLE
Sport - Parse match stats for rugby

### PR DESCRIPTION
**Pull request by mchv**
This is a first part of my work to try to add match statistics for rugby games during the world cup.
The associated changes are:
- Upgrade to latest scala `2.11.x` version
- Fix unchecked warning due to erasure
- Do not add `optimise` argument to scalac, due to [a scala compiler bug](https://issues.scala-lang.org/browse/SI-6782) generating a warning

```
warning: At the end of the day, could not inline @inline-marked method augmentString
```

Please not that `optimize` in scala `2.11.x` is considered a bit broken, and people recommend against using it, see [akka recommendation](http://doc.akka.io/docs/akka/snapshot/intro/getting-started.html#Do_not_use_-optimize_Scala_compiler_flag).

Scala 2.12 will use [by default a new optimizer](http://magarciaepfl.github.io/scala/).
- Update the model and parse player stats

The parsing (`parseLiveEventsStatsToGetMatchStat`) is not called apart from the test, so it should have currently no effect on `PROD`.

The next step is to parse the team stats and then display some of them using same football stats UI.

### CODE REVIEW

**Comment by jennysivapalan** [comment]
I think it would be best to separate the scala upgrade to a separate PR. If anyone disagrees please shout as don't want to add more work for @mchv.

**Comment by mchv** [comment]
@jennysivapalan https://github.com/guardian/frontend/pull/10438

File: project/Prototypes.scala [hunk 1]
```diff
@@ -16,11 +16,11 @@ trait Prototypes {
     organization := "com.gu",
     maxErrors := 20,
     javacOptions := Seq("-g","-encoding", "utf8"),
-    scalacOptions := Seq("-unchecked", "-optimise", "-deprecation", "-target:jvm-1.8",
```
Comment (gklopper) [inline project/Prototypes.scala]: is `-optimise` deprecated in 2.11.7?

Comment (gklopper) [inline project/Prototypes.scala]: Sorry, I see your comment in the description...

**Comment by gklopper** [comment]
:+1:

**Comment by mchv** [comment]
I will rebase those changes on https://github.com/guardian/frontend/pull/10463 changes as they are more player statistics that I plan to use later (team stats first).

**Comment by jennysivapalan** [comment]
@mchv shall we close this?

**Pull request closed without merging**